server/data/restClient.ts

```diff
@@ -12,7 +12,6 @@ interface GetRequest {
   query?: object | string
   headers?: Record<string, string>
   responseType?: string
-  raw?: boolean
```

Comment:
dependency bump made typescript unhappy that we were returning a union of types... we weren't actually using the raw response anywhere so I've removed it